api/models.py

```diff
@@ -1,8 +1,61 @@
 from django.db import models
 
-# 모델링 과제를 이곳에서 해주시면 됩니다! (주석은 나중에 지우셔도 돼요!)
 
 # [제약조건]
 # 1. 1:1과 1:n의 관계 포함
 # 2. 각 모델에 필드 최소 3개 이상 포함
-# 3. 서비스 관련 모델 3개 이상 + 유저 모델 1개 구현 (단, 유저는 필수 아님)
\ No newline at end of file
+# 3. 서비스 관련 모델 3개 이상 + 유저 모델 1개 구현 (단, 유저는 필수 아님)
+
+
+# 하나의 영화관(db : megabox)에서 상영되는 영화, 누적관객 수, 상영날짜 등으로 모델링
+
+class Movie(models.Model):
+    title = models.CharField(max_length=200, primary_key=True)
+    director = models.CharField(max_length=100)
+    release_year = models.IntegerField()
+    running_time = models.IntegerField()  # minutes 단위
+    country = models.CharField(max_length=100)
+    genre = models.CharField(max_length=100)
+
+    def givetitle(self):
+        return self.title
+
+    def __str__(self):
+        return self.title
+
+
+class ScreeningDates(models.Model):  # 시기별 누적 관객수
+    title = models.ForeignKey('Movie', on_delete=models.CASCADE)
+    start_date = models.DateTimeField()
+    finished_date = models.DateTimeField(null=True)
+    total_audience = models.IntegerField()
+
+    def __str__(self):
+        return str(self.title)
+
+
+class GenreStat(models.Model):  # 영화 장르별 통계
```

Comment:
GenreStat의 모델을 만들기 보다는 아래와 같이 Genre 모델을 새로 만들어 Movie와 연결시켜 준 후 사용하는 편이 더 나아보입니다!(위에서는 movie의 genre 필드를 choice 형태로 만들기를 추천드렸지만, 이 모델을 바꿔 사용한다면 ForeignKey로 참조하는 편도 좋을 것 같아요🥝🍈)
 count와 같은 값은 django orm을 사용하면 쉽게 참조할 수 있어요! (`Movie.objects.filter(genre='SF').count()`)

```suggestion
class Genre(models.Model): 
    name = models.CharField(max_length=20)

class Movie(models.Model):
    ...
    genre = models.ForeignKey('Genre', on_delete=models.SET_DEFAULT, default="SF", related_name='movies')
```